ajax/libs/fomantic-ui/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "fomantic-ui",
+  "filename": "semantic.min.js",
+  "description": "Community fork of the popular Semantic-UI framework",
+  "keywords": [
+    "semantic",
+    "fomantic",
+    "framework",
+    "ui"
+  ],
+  "author": "hammy2899",
```

Comment:
Maybe also Jack Lukic? Refer https://github.com/fomantic/Fomantic-UI/blob/master/package.json#L7

Reply:
Jack has no involvement with FUI.

Reply:
He had? https://github.com/fomantic/Fomantic-UI/graphs/contributors

Reply:
@extend1994 Any news?

Reply:
Sorry for the late reply, missed this one. 
Didn't know the situation you described. Both are fine to me :)